packages/camera/example/test_driver/super_camera.dart

```diff
@@ -0,0 +1,381 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
These tests will only be used locally, since there isn't a camera for an iOS simulator/android emulator.